_posts/2020-08-13-convertextract-app.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: post
+title:  "How to use the new Convertextract application for 'quality control' of ELAN annotations"
+author: fineen
+categories: [ Tutorial, Blog ]
+tags: [ basic ]
+image: assets/images/gui.png
+description: "How to install and use the new Convertextract app for 'quality control' purposes"
+featured: false
+hidden: false
+---
+
+# TL;DR
+
+Have you ever wanted to *NOT* spend hours tediously checking that **k + '** is written as **k̓** and not **k'**?
+If you said **YES!**, *Convertextract* is the app for you. With minimal technical knowledge, you can now systemically make your ELAN annotations consistent.
+
+# What you need to know to understand this post
+
+I assume that you know some background about the [g2p library](https://github.com/roedoejet/g2p), but if your language already has the desired template for conversion (mapping), then you don't need to worry too much about this.
```

Comment:
* Careful not to describe/define field specific jargon term with another (i.e. conversion & mapping).
* What is the *desired* template? Important to know if my language already has it. 
* Even though this section is to set what the prerequisite knowledge is, I think it's more useful if this spot is one of the most simply put, explanatory sections. If I'm from a language community and I want to know if this is something I should start to learn about, this section should explain to me what I should know to move forward
    * What is G2P?What it stands for and explained in a brief half a sentence.